pages/learn/manage/device-statuses.md

```diff
@@ -11,20 +11,21 @@ The application overview page shows the status of each device in the application
-| Offline           | The device is offline and is not connected to VPN and has not any recent API communications.                                                             |
-| Post Provisioning | The device has been [provisioned][device-provisioning] but has not yet come online.                                                                      |
-| Inactive          | The device has been [deactivated][deactivated] or has been [preregistered][preregistered] but has not yet connected to the {{ $names.cloud.lower }} API. |
+| Status            | Description                                                                                                                                                     |
+|-------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Online            | The device is online and is connected to the VPN or has recent API communications.                                                                              |
+| Configuring       | The device is applying OS configuration.                                                                                                                        |
+| Updating          | The device is updating to a new application release.                                                                                                            |
+| Offline           | The device is offline and is not connected to VPN and has not any recent API communications.                                                                    |
+| Post Provisioning | The device has been [provisioned][device-provisioning] but has not yet come online.                                                                             |
+| Inactive          | The device has been [deactivated][deactivated] or has been [preregistered][preregistered] but has not yet connected to the {{ $names.cloud.lower }} API.        |
+| Frozen            | The device has been frozen because it's outside the organization's allowance, or is in a paid [application type][application type] on a free tier organization. |
 
 ## Device connectivity
 
 ![Device connectivity indicators](/img/common/main_dashboard/device_status.png)
 
-In addition to the device status, there are indicators to show if the device is currently connected to the VPN (checkmark icon) and has recent API communications (heartbeat icon). These indicators can be used to identify common issues, such as a firewall blocking VPN traffic or an inability of the device to communicate with the {{ $names.cloud.lower }} API.
+In addition to the device status, there are indicators to show if the device is having any partial connectivity issue. These indicators can be used to identify common issues, such as a firewall blocking VPN traffic or an inability of the device to communicate with the {{ $names.cloud.lower }} API.
```

Comment:
I think the previous version that explained each condition separately is more informative. What do I miss?

Reply:
I essentially just dropped the references to the two separate vpn/heartbeat status icons that we no longer have.
Now the secondary status indicator explains what connectivity issue there is (as seen in the new image).

Perhaps this could be further improved by changing the example below to enumerate all possible cases, but I think doesn't need to be part of this PR.

Reply:
I didn’t realise we changed that and also missed the image! Thanks for clarifying.